packages/form/src/components/TextAreaField/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import { describe, expect, jest, test } from '@jest/globals'
+import { screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { TextAreaField } from '..'
+import { Submit } from '../..'
+import {
+  renderWithTheme,
+  shouldMatchEmotionSnapshotFormWrapper,
+} from '../../../../.jest/helpers'
+import { mockErrors } from '../../../mocks'
+import { Form } from '../../Form'
+
+describe('TextAreaField', () => {
+  test('should render correctly', () =>
+    shouldMatchEmotionSnapshotFormWrapper(
+      <TextAreaField label="Test" name="test" />,
+    ))
+
+  test('should render correctly generated', async () => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const onSubmit = jest.fn<any>()
```

Comment:
Doesn't it work like this ?

```suggestion
    const onSubmit = jest.fn(() => {})
```

So we can remove the eslint disable